manifests/04-deployment-capi.yaml

```diff
@@ -85,14 +85,16 @@ spec:
           value: "0.0.1-snapshot"
         - name: METRICS_PORT
           value: "9193"
+        - name: METRICS_BIND_ADDRESS
+          value: "127.0.0.1"
```

Comment:
For the reader, we want port 9193 to be internal only by default.
RBAC proxy will take care of proxying requests received from outside the pod on port 9194 to port 9193.